js/views/itemVw.js

```diff
@@ -170,10 +168,11 @@ module.exports = baseVw.extend({
     var self = this,
         buyModel = new Backbone.Model();
     buyModel.set(this.model.attributes);
-    this.buyWizardView = new buyWizardVw({model:buyModel, parentEl: '#modalHolder', userModel: this.options.userModel});
-    this.subViews.push(this.buyWizardView);
-    this.subModels.push(buyModel);
-    $('#obContainer').addClass('overflowHidden').addClass('blur');  
+    this.buyWizardView = new buyWizardVw({model:buyModel, userModel: this.options.userModel});
+    this.registerChild(this.buyWizardView);
+    $('#modalHolder').html(this.buyWizardView.el); //add to DOM first, or accordion will have zero width when initialized
+    this.buyWizardView.render().el;
+    $('#obContainer').addClass('overflowHidden').addClass('blur');
```

Comment:
I know that you didn't introduce this line, but if you're in there, the classes here can be combined into one addClass call.